plugins/woocommerce/tests/e2e-pw/tests/merchant/create-page.spec.js

```diff
@@ -23,8 +23,12 @@ test.describe(
 
 			const canvas = await getCanvas( page );
 
+			// TODO (Gutenberg 19.9): Remove this click() step.
+			// Current stable version of Gutenberg (19.7) doesn't show the "Empty block" element right away, you need to click on the "Add default block" element first before it appears.
+			// Upcoming Gutenberg nightly (version 19.9) no longer shows the "Add default block" element, but rather displays the "Empty block" right away.
+			// There's no need for this click() step anymore when GB 19.9 comes out.
```

Comment:
I'll create a separate GH issue for this TODO item after this PR is merged so that we don't forget it. 

Same with the other `TODO (Gutenberg 19.9)` item in `plugins/woocommerce/tests/e2e-pw/utils/editor.js` above.